com.unity.postprocessing/PostProcessing/Runtime/Utils/RuntimeUtilities.cs

```diff
@@ -862,14 +862,24 @@ public static bool isAndroidOpenGL
         /// Returns <c>true</c> if the target platform is WebGL,
         /// <c>false</c> otherwise.
         /// </summary>
-        public static bool isWebGL
+        public static bool isWebNonWebGPU
         {
-            get {
-#if UNITY_WEBGL
-                return true;
+            get
+            {
+#if UNITY_EDITOR
+    #if UNITY_WEBGL
+                return PlayerSettings.GetGraphicsAPIs(BuildTarget.WebGL)[0] != GraphicsDeviceType.WebGPU;
+    #else
+                                                            return false;
+    #endif
```

Comment:
```suggestion
#if UNITY_EDITOR
    #if UNITY_WEBGL
        #if UNITY_2023_2_OR_NEWER
                return PlayerSettings.GetGraphicsAPIs(BuildTarget.WebGL)[0] != GraphicsDeviceType.WebGPU;
        #else
                return true;
        #endif
    #else
                return false;
    #endif
```

`GraphicsDeviceType.WebGPU` check not supported on old editors.

Reply:
you're right total miss by me thanks! I'll run the tests again.